src/View/Helper/FormSelect.php

```diff
@@ -185,7 +185,7 @@ public function renderOptions(array $options, array $selectedOptions = []): stri
                 $value = $optionSpec['value'];
             }
             if (isset($optionSpec['label'])) {
-                $label = $optionSpec['label'];
+                $label = (string) $optionSpec['label'];
```

Comment:
The need for added `(string)` casts is a bit suspicious - what happened here?

Reply:
For example the `year` part of the `FormMonthSelect` Element returns an int as value. Where should I cast the value to string? Psalm complains that the Translator dont want an int.